### PR TITLE
Add Ruby 2.4.1 to test matrix

### PR DESCRIPTION
**Pull request by sgerrand**
💁  These changes add Ruby 2.4.1 to test matrix's Ruby versions.

### CODE REVIEW

File: spec/fixtures/rails42.rb [hunk 1]
```diff
@@ -10,6 +10,7 @@ class Dummy < Rails::Application
   config.secret_key_base = '2624599ca9ab3cf3823626240138a128118a87683bf03ab8f155844c33b3cd8cbbfa3ef5e29db6f5bd182f8bd4776209d9577cfb46ac51bfd232b00ab0136b24'
   config.session_store :cookie_store, key: '_rails32_session'
 
+  config.log_level = :info
```
Comment (sgerrand) [inline spec/fixtures/rails42.rb]: 🙋 Is this needed?

File: Appraisals [hunk 2]
```diff
@@ -2,8 +2,8 @@ appraise 'rails32' do
   gem 'rails', '~> 3.2.0'
 end
 
-appraise 'rails40' do
-  gem 'rails', '~> 4.0.0'
+appraise 'rails42' do
+  gem 'rails', '~> 4.2.0'
```
Comment (sgerrand) [inline Appraisals]: Seems like the easier approach is to have a build matrix and exclude incompatible Rails and Ruby versions on a case by case basis

Comment (sgerrand) [reply Appraisals]: i.e. this is a change which falls outside the scope of this pull request.

File: spec/fixtures/rails42.rb (hunk 1 above)
Comment (timstott) [reply spec/fixtures/rails42.rb]: Rails 4.2 will raise a warning when `log_level` isn't specified in production 🍦

**Comment by timstott** [comment]
I'm a little confused as to why this was closed. It would be nice to support the latest Ruby version. I understand a test matrix would be preferable but for the time being this works reasonably well.

**Comment by sgerrand** [comment]
Your pull request description closed it! Be wary of using words like "fix" next to a pull request or issue reference.

**Comment by sgerrand** [comment]
✨ I'll revisit this myself later.

**Comment by timstott** [comment]
@sgerrand Oh sorry about that. I didn't know `fix` in PR descriptions worked too 😬.

**Comment by bliof** [comment]
Closing in favour of https://github.com/FundingCircle/loga/pull/104

**Pull request closed without merging**